hyrax.gemspec

```diff
@@ -82,7 +82,7 @@ SUMMARY
 
   # temporary pin to 2.17 due to failures caused in 2.18.0
   spec.add_development_dependency "capybara", '~> 2.4', '< 2.18.0'
-  spec.add_development_dependency 'capybara-maleficent', '~> 0.2'
+  spec.add_development_dependency 'capybara-maleficent', '~> 0.3'
```

Comment:
Can we be more strict about this?

It seems like `~> 0.x` is going to cause the build to break on future releases. Perhaps `~> 0.3.0`, or even `0.3`